src/components/PopupWithConfirm.js

```diff
@@ -25,19 +25,34 @@ export default class PopupWithConfirm extends Popup {
   }
 
   close() {
-    this._handleFormSubmit.reset();
     super.close();
-    this._popupForm.removeEventListener("submit", (e) => {
-      e.preventDefault();
-      this._handleFormSubmit(e);
-    });
+    this._popupForm.removeEventListener("submit", this._handleFormSubmit);
```

Comment:
Please, delete method `close` because you don't need it anymore. The listener you are trying to remove in line 29 doesn't exist. You do it correctly in `_removeEventListeners`